src/ResourceManager/Insights/Commands.Insights/help/Add-AzureRmLogProfile.md

```diff
@@ -22,10 +24,9 @@ Add-AzureRmLogProfile -Name <String> [-StorageAccountId <String>] [-ServiceBusRu
 The **Add-AzureRmLogProfile** cmdlet creates a log profile.
 
 ## EXAMPLES
-
-### 1:
-```
-
+### Example 1 : Add a new log profile to export the activity log matching the location condition to a storage account 
+```yaml
+Add-AzureRmLogProfile -Locations "Global,West US" -Name ExportLogProfile -StorageAccountId /subscriptions/40gpe80s-9sb7-4f07-9042-b1b6a92ja9fk/resourceGroups/activitylogRG/providers/Microsoft.Storage/storageAccounts/activitylogstorageaccount 
```

Comment:
> -Locations "Global,West US"

Replace to:
-Locations global,westus

Or:

-Locations "Global", "West US"

I prefer the lower case scenario because that is what the API returns, but either work.